scalableminds/webknossos
# Hide login in navbar when corresponding feature flag is enabled

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- enable flag and disable auto-login
- open wk and see that there is no form in the navbar

### Issues:
- fixes #3188 

------
